spikingjelly/activation_based/neuron/few_spike.py

```
@@ -0,0 +1,589 @@
+    spike sequence with shape ``[K, ...]``. Multi-step mode does not aggregate the
+    output sequence.
+
+    Inputs must be floating point tensors. Coding table parameters are registered
+    as buffers and follow module ``to(device/dtype)`` conversions. If the input
+    dtype differs from the buffer dtype, PyTorch dtype promotion rules apply.
+    """
+
+    def __init__(
+        self,
+        table: FewSpikeTable,
+        surrogate_function: surrogate.SurrogateFunctionBase = surrogate.Sigmoid(),
```

Ruff (0.15.18)

[warning] Do not perform function call `surrogate.Sigmoid` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)